### PR TITLE
Readme Updates; Web Sequence Diagram; Config Panel Text

### PR DESCRIPTION
**Pull request by awatson1978**
Hi.  First batch of updates.  Cleaned up through the configuration panel.  I'm pausing here before proceeding with the Auth workflow, because there are a few key refactoring I want to do that could break things (ex:  renaming `MeteorOAuth2Server` to `MyOAuthService`).  So submitting this for review before moving onto those things.  

ps.  I tried to submit this to a feature branch, but it's not created, so have to PR against `master`.  Sorry about that.

pps.  Also, my apologies for this sudden burst of activity without an introduction.  I've got a client that has me on the clock, and I want to hand off a version of this repository to a coworker by next week.  Trying to do a once-through to make sure everything is working and documented before handing it over; then things will slow down a bit as we move into a somewhat slower-paced feature development cycle.

### CODE REVIEW

**Comment by awatson1978** [comment]
Something is broken.  Rolling back.

File: examples/clientApplication/client/index.html [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
 
 <template name="main">
     {{> loginButtons}}
 
-    <p>For the sake of convience while you experiment, you can reset the MeteorOAuth2Server configuration.</p>
-    <button class="resetServiceConfiguration">Reset service configuration</button>
-</template>
\ No newline at end of file
+    <p>For the sake of convience while you experiment, you can reset configuration for the ClientApplication.</p>
+    <button class="resetServiceConfiguration login-button">Reset service configuration</button>
```
Comment (vangorra) [inline examples/clientApplication/client/index.html]: login-button class is unrelated to the functionality of the button.

File: examples/resourceServer/client/index.html [hunk 2]
```diff
@@ -24,7 +24,7 @@
     {{#unless isUrlParamsValid }}
         <p>The URL params are not set in order to authorize an oauth2 token.</p>
         <p>The following link will simulate arriving to this page from another site that wishes to authenticate with oauth2.</p>
-        <a href="/?client_id=resourceServerClient&redirect_uri=http://localhost:3200/_oauth/MeteorOAuth2Server&response_type=code">Simulate proper link.</a>
+        <a href="/?client_id=clientApplication&redirect_uri=http://localhost:3200/_oauth/MeteorOAuth2Server&response_type=code">Simulate proper link.</a>
```
Comment (vangorra) [inline examples/resourceServer/client/index.html:27]: Thanks for fixing this.

File: examples/resourceServer/server/rest.js [hunk 3]
```diff
@@ -16,6 +16,8 @@ JsonRoutes.Middleware.use(
 );
 
 /**
+ * OAUTH FLOW - A7.2
```
Comment (vangorra) [inline examples/resourceServer/server/rest.js:19]: Matching parts of the examples with the workflow diagram. Nice touch!

**Comment by vangorra** [comment]
Take care of the unnecessary class on the button and I'll merge. 

Also, open an issue so we can properly keep a dialog about the refactors you are proposing.

File: examples/resourceServer/server/rest.js (hunk 3 above)
Comment (awatson1978) [inline examples/resourceServer/server/rest.js:19]: That's the idea.  I'm trying to go through the whole thing and match up code with diagram, and vice-versa.

**Comment by awatson1978** [comment]
Okay, login-button class is removed.  Going to start on the ResourceServer next.

**Pull request merged**